volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1504,39 +1500,40 @@ def is_valid(self):
         )
 
     def _is_kernel_prior_to_struct_mount(self) -> bool:
-        """Helper to distinguish between kernels prior to version 3.3.8 that
-        lacked the 'mount' structure and later versions that have it.
+        """Helper to distinguish between kernels prior to version 3.3 which lacked the
+        'mount' struct, versus later versions that include it.
+        See 7d6fec45a5131918b51dcd76da52f2ec86a85be6.
 
-        The 'mnt_parent' member was moved from struct 'vfsmount' to struct
-        'mount' when the latter was introduced.
+        # Following that commit, also in kernel version 3.3 (3376f34fff5be9954fd9a9c4fd68f4a0a36d480e),
+        # the 'mnt_parent' member was relocated from the 'vfsmount' struct to the newly
+        # introduced 'mount' struct.
 
         Alternatively, vmlinux.has_type('mount') can be used here but it is faster.
```

Comment:
Just spotted this no "alternative blah can be used *BUT* it is faster"?  Why wouldn't we use it if it's faster?  Either it needs to be this, or something about this needs changing...

Reply:
Yeah, it looks like I was thinking about getting vmlinux to check that. I probably wasn't aware of `context.symbol_space` at the time, or it just didn't come to mind. Since `get_module_from_volobj_type()` is more expensive than `self.has_member("mnt_parent")` but technically produces the same result, I chose the latter.

I will update this here to unblock it, but I still feel it's not directly related to the goal or changes in this PR

Reply:
Oh, it was more just the text, "but it was faster, but..."

Reply:
hah sorry, I changed it anyway... now it doesn't need the clarification so it also fixed the typo :)